vignettes/aedseo_introduction.Rmd

```diff
@@ -174,7 +174,7 @@ In the following figure, the simulated data (solid circles) is visualized alongs
 plot(tsd_data)
 ```
 
-Next, the time series data object is passed to the `aedseo()` function. Here, a window width of $k=5$ is specified, meaning that a total of 5 weeks is used in the local estimate of the exponential growth rate. Additionally, a 95\% confidence interval is specified. Finally, the exponential growth rate is estimated using quasi-Poisson regression to account for overdispersion in the data.
+Next, the time series data object is passed to the `aedseo()` function. Here, a window width of $k=5$ is specified, meaning that a total of 5 weeks is used in the local estimate of the exponential growth rate. $na_percentage_allowed = 0.4$ defines how many percentage of the observables in the k window that are allowed to be NA, here 0.4*5 = 20%. Additionally, a 95\% confidence interval is specified. Finally, the exponential growth rate is estimated using quasi-Poisson regression to account for overdispersion in the data.
```

Comment:
Consider calling it window "size" instead of  window "width" here. "size" is used in the function documentation.

Reply:
Also check this section for the "percentage" vs "fraction" comment